.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,19 @@
+name: "Chromatic"
+on: push
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: "10.16.3"
```

Comment:
Note: a very specific version of node is required, or the build won't build.

Reply:
This restriction is set in the root `package.json`:
https://github.com/cultureamp/kaizen-design-system/blob/master/package.json#L7-L9